conf/app.routes

```diff
@@ -3,4 +3,7 @@
 PUT         /services/:id/status          uk.gov.hmrc.customsservicestatus.controllers.CustomsServiceStatusController.updateServiceStatus(id: String)
 GET         /services                     uk.gov.hmrc.customsservicestatus.controllers.CustomsServiceStatusController.list()
 GET         /services/planned-work        uk.gov.hmrc.customsservicestatus.controllers.AdminCustomsServiceStatusController.getAllPlannedWorks()
-POST        /services/messages            uk.gov.hmrc.customsservicestatus.controllers.AdminCustomsServiceStatusController.updateWithOutageData()
\ No newline at end of file
+POST        /services/messages            uk.gov.hmrc.customsservicestatus.controllers.AdminCustomsServiceStatusController.updateWithOutageData()
+GET         /outages                    uk.gov.hmrc.customsservicestatus.controllers.AdminCustomsServiceStatusController.findAllOutages()
+GET         /outages/:id                uk.gov.hmrc.customsservicestatus.controllers.AdminCustomsServiceStatusController.findOutage(id: UUID)
+GET         /outages/archive/:id         uk.gov.hmrc.customsservicestatus.controllers.AdminCustomsServiceStatusController.archiveOutage(id: UUID)
```

Comment:
Discussed offline and agreed this should be a PUT rather than a GET as it is updating an outage on the server to archive it.

Reply:
minor: would also be good to vertically align the routes here.